packages/auth/test/auth-utils.spec.ts

```diff
@@ -428,7 +427,7 @@ describe('Auth utilities', () => {
         );
       });
 
-      test('should fail on verify the token and return anonymous users', async () => {
+      test('should fail on verify the token and return anonymous users 2', async () => {
```

Comment:
??

Reply:
Eslint complains about tests with same name. There were two tests with same name in this file.

Reply:
Let's reword this one, actually the test does `should verify the token and return a remote user`. I think that's the right description here.

Reply:
Sure. I was't sure what should go there. Thanks.